test/integration/targets/rds_param_group/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+rds_param_groups:
+  - name: rds-test-param-group-pg-96
```

Comment:
Resource names should incorporate the `resource_prefix` variable, as is done for tests like [ec2_group](https://github.com/ansible/ansible/blob/devel/test/integration/targets/ec2_group/defaults/main.yml#L3).

There are two reasons for this:

1) It avoids resource collision when multiple test runs occur simultaneously.
2) It makes it possible to identify which test run leaked resources during a test.